english/you-are-in-minecraft/minecraft.md

```diff
@@ -0,0 +1,17 @@
+You open your eyes and there you are surrounded by a crafting table, a furnace, and what seems to be a new addition to minecraft, marshmallow walls.
```

Comment:
It looks like there is no link to this new file.  Did you forget to commit that change?  Make sure you create (and commit) an option for each new "page" you create.